app/controllers/sessions_controller.rb

```diff
@@ -14,9 +14,14 @@ def create
     end
   end
 
+  def update
+    user_info = request.env["omniauth.auth"]
```

Comment:
Had to make the env lower cased for this to work. Gave us a little trouble because we did not fully understand why that had to be.